src/Argument/Argument.php

```diff
@@ -3,6 +3,7 @@
 namespace League\CLImate\Argument;
 
 use function is_array;
+use League\CLImate\Exceptions\InvalidArgumentCastTypeException;
```

Comment:
Classes should be imported before functions

Reply:
Didn't knew that. Okay.